cli-commands/vm/post/show.rb

```diff
@@ -1,14 +1,14 @@
 # frozen_string_literal: true
 
 UbiRodish.on("vm").run_on("show") do
-  fields = %w[id name state location size unix_user storage_size_gib ip6 ip4_enabled ip4 private_ipv4 private_ipv6 subnet firewalls].freeze.each(&:freeze)
-  firewall_fields = %w[id name description location path firewall_rules].freeze.each(&:freeze)
-  firewall_rule_fields = %w[id cidr port_range].freeze.each(&:freeze)
+  fields = %w[id name state location size unix-user storage-size-gib ip6 ip4-enabled ip4 private-ipv4 private-ipv6 subnet firewalls].freeze.each(&:freeze)
+  firewall_fields = %w[id name description location path firewall-rules].freeze.each(&:freeze)
+  firewall_rule_fields = %w[id cidr port-range].freeze.each(&:freeze)
 
   options("ubi vm location/(vm-name|_vm-ubid) show [options]", key: :vm_show) do
     on("-f", "--fields=fields", "show specific fields (default: #{fields.join(",")})")
-    on("-r", "--firewall_rule_fields=fields", "show specific fields (default: #{firewall_rule_fields.join(",")})")
-    on("-w", "--firewall_fields=fields", "show specific fields (default: #{firewall_fields.join(",")})")
+    on("-r", "--rule-fields=fields", "show specific firewall rule fields (default: #{firewall_rule_fields.join(",")})")
```

Comment:
I think my inclination on something like this would be to maintain the `firewall` prefix to this. There might not be other rules presently, but that might not always be true and regardless I like the clarity/explicitness. That said, it's certainly more verbose, and that's always a tension in these things. I tend to err on the side of correctness/specificity, even at the cost of being more verbose, but I know others differ on where their tastes lie.

Reply:
Ah, that makes sense. Definitely will want to work within those constraints.